### PR TITLE
RTF files in bulk import

### PR DESCRIPTION
**Pull request by actlikewill**
- [x]  check for rtf files in importer
- [x] import all other files as media attachments

### CODE REVIEW

File: peachjam/resources.py [hunk 1]
```diff
@@ -96,21 +97,35 @@ def clean(self, value, row=None, **kwargs):
...
+
+        if docx_urls:
+            return docx_urls[0]
+        if rtf_urls:
+            return rtf_urls[0]
+        if pdf_urls:
+            return pdf_urls[0]
 
-        return source_url
+        return source_files[0]
```
Comment (longhotsummer) [inline peachjam/resources.py:128]: Will source_files always have at least one element?

File: peachjam/resources.py [hunk 2]
```diff
@@ -86,7 +87,7 @@ def clean(self, value, row=None, **kwargs):
                     elif mime in DOC_MIMETYPES:
```
Comment (longhotsummer) [inline peachjam/resources.py:87]: Does DOC_MIMETYPES need to be updated to include RTF?

File: peachjam/resources.py (hunk 1 above)
Comment (actlikewill) [reply peachjam/resources.py:128]: Yes it will

File: peachjam/resources.py (hunk 2 above)
Comment (actlikewill) [reply peachjam/resources.py:87]: I've added it. I thought that we would have to create a separate pipline for .rtf but adding the mimetype now extracts the text correctly. :100:

**Review by longhotsummer — APPROVED** [review]
I hope it's tested and works :)

**Comment by longhotsummer** [comment]
Fixed #801

**Pull request merged**